dataset_loaders/data_augmentation.py

```diff
@@ -598,22 +658,26 @@ def random_transform(x, y=None,
         if y is not None and len(y) > 0:
             y = y.transpose(pattern)
             y = y[..., top:top+crop[0], left:left+crop[1]]
+        if flow is not None:
+            flow = flow.transpose(pattern)
+            flow = flow[..., top:top+crop[0], left:left+crop[1]]
```

Comment:
There is no need to do this if you concatenate the flow `before` computing the crop / pad etc